src/samples/HelloWorld/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.swiftinterface*
```

Comment:
Maybe this .gitignore should go in the src/samples directory (or even higher up?)

Reply:
Moved to the global `.gitignore`.